docs/en/setup/envoy/examples/metrics/docker-compose.yaml

```diff
@@ -17,7 +17,7 @@
 version: "3"
 services:
   envoy:
-    image: envoyproxy/envoy-alpine:latest
+    image: envoyproxy/envoy-alpine:v1.16.2
```

Comment:
Envoy now doesn't have a latest tag, and pinning the image version is a good practice IMO.